<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>757</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="typeLabel">
          <property name="font">
           <font>
            <family>Segoe UI Historic</family>
            <pointsize>15</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Режим работы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="isTypeBox">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <item>
           <property name="text">
            <string>Разовый запуск</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Работа по таймеру</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="updateLabel">
            <property name="font">
             <font>
              <family>Segoe UI Historic</family>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Таймер (сек.)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="timerSpin"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="strtBtn">
      <property name="font">
       <font>
        <family>Segoe UI Historic</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="deleteLabel">
        <property name="font">
         <font>
          <family>Segoe UI Historic</family>
          <pointsize>15</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Удалять входной файлы после обработки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="isDeleteBox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <item>
         <property name="text">
          <string>Да</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Нет</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="matchLabel">
        <property name="font">
         <font>
          <family>Segoe UI Historic</family>
          <pointsize>15</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>При совпадении имени файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="isMatchBox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <item>
         <property name="text">
          <string>Перезаписать</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Добавить суффикс</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="sourceLabel">
        <property name="font">
         <font>
          <family>Segoe UI Historic</family>
          <pointsize>15</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Исходная директория</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sourcePath"/>
      </item>
      <item>
       <widget class="QToolButton" name="sourceDir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="bytesLabel">
        <property name="font">
         <font>
          <family>Segoe UI Historic</family>
          <pointsize>15</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Значение для операции (8 байт)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bytesValue"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="resultLabel">
          <property name="font">
           <font>
            <family>Segoe UI Historic</family>
            <pointsize>15</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Путь для сохранения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="resultPath"/>
        </item>
        <item>
         <widget class="QToolButton" name="resultDir">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="maskLabel">
          <property name="font">
           <font>
            <family>Segoe UI Historic</family>
            <pointsize>15</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Маска файлов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maskEdit">
          <property name="placeholderText">
           <string>По умолчанию: *.*</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
